StereoKitC/stereokit.h

```diff
@@ -1741,6 +1741,7 @@ SK_API bool32_t platform_keyboard_get_force_fallback(void);
 SK_API void     platform_keyboard_set_force_fallback(bool32_t force_fallback);
 SK_API void     platform_keyboard_show              (bool32_t visible, text_context_ type);
 SK_API bool32_t platform_keyboard_visible           (void);
+SK_API bool32_t platform_keyboard_switch(text_context_ type, char** keyboard_layout, int layouts_num);
```

Comment:
I would still prefer to see this as `platform_keyboard_set_layout`/`Platform.KeyboardSetLayout`, unless you have some justification for why `switch` is a better name?